odoo/addons/base/migrations/12.0.1.3/noupdate_changes.xml

```diff
@@ -1,17 +1,174 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
-  <record id="gt" model="res.country">
-    <field name="currency_id" ref="GTQ"/>
```

Comment:
Isn't this country finally assigned that currency?

Reply:
Yes, but was also assigned in v11, so obviously disappear now from here.